srcpkgs/kbfs/template

```diff
@@ -1,23 +1,28 @@
 # Template file for 'kbfs'
 pkgname=kbfs
-version=2.11.0
+version=4.4.1
 revision=1
+wrksrc="client-${version}"
 build_style=go
-go_import_path="github.com/keybase/kbfs"
-go_package="${go_import_path}/kbfsfuse ${go_import_path}/kbfsgit/git-remote-keybase ${go_import_path}/kbfstool ${go_import_path}/redirector"
-go_build_tags="production"
-depends="keybase util-linux"
+go_import_path="github.com/keybase/client/go/kbfs"
+go_package="${go_import_path}/kbfsfuse
+${go_import_path}/kbfsgit/git-remote-keybase
+${go_import_path}/kbfstool ${go_import_path}/redirector"
+makedepends="git"
+depends="keybase fuse"
 short_desc="Keybase Filesystem"
-maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
+maintainer="itamiko <itamiko@pm.me>"
 license="BSD-3-Clause"
 homepage="https://keybase.io/docs/kbfs"
-distfiles="https://github.com/keybase/kbfs/archive/v${version}.tar.gz"
-checksum=0e97cab1c90592035ec6763e22c78c08a95dd3467419d15b9e45b3d0887b594d
+distfiles="https://github.com/keybase/client/archive/v${version}.tar.gz"
+checksum=dfac5ff4ecd5cd75f22ebece24c2e074cc83e38c7b207e074ca2c96ec3c96b7b
+
+do_configure() {
+	export GOPATH="${wrksrc}/.gopath"
+	vlicense LICENSE
```

Comment:
Why do you override `do_configure`? What is the issue with the one provided by the build_style? Also vlicense should stay in `post_install`